app/assets/javascripts/constants/api_constants.es6.jsx

```diff
@@ -65,8 +65,8 @@
     get rooms() {
       return {
         show: (id) => `/api/rooms/${id}`,
-        index: (page=1, conference_id=1) => {
-          return `/api/rooms?page=${page}&conference_id=${conference_id}`;
+        index: (conference_id=1) => {
```

Comment:
What happens if we remove the `=1` now? Hopefully we should always be having a valid `conference_id` now.